src/main/include/subsystems/SubSystemConfig.h

```diff
@@ -72,4 +72,12 @@ const double LAUNCHER_ZERO_OFFSET = 13.501;
 const double MAX_PIVOT_ANGLE = 75.0;
 const double MIN_PIVOT_ANGLE = 5.0;
 
-const bool INVERT_INDEXER = false;
\ No newline at end of file
+const bool INVERT_INDEXER = false;
+// ###########################################################
+// #                         Climber                         #
+// ###########################################################
+
```

Comment:
add a flag to invert the main motor (i.e. the left climber motor)

Reply:
done